aws/hll/aws-s3-transfer-manager/src/download/handle.rs

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// Shared handle used across a single download request
+#[derive(Debug, Clone)]
+pub(super) struct DownloadHandle {
+    pub(crate) client: aws_sdk_s3::Client,
```

Comment:
Does this need to be `Arc` so that the clone is cheap? Do we already have an `Arc` inside the client? I can't recall.

Reply:
I believe clients are already cheap to clone. Will double check...

indeed they seem to be
```rust
#[derive(::std::clone::Clone, ::std::fmt::Debug)]
pub struct Client {
    handle: ::std::sync::Arc<Handle>,
}
```